server.go

```diff
@@ -1680,6 +1683,29 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 	return s, nil
 }
 
+// UpdateEstimatorValue is a callback function to change estimator value in the  
+// main cfg when the lncli setmccfg is called to change the estimator value. 
+ func (s *server) UpdateEstimatorValue(estimator routing.Estimator) {
+	switch c := estimator.Config().(type) {
+	case routing.AprioriConfig:
+		s.cfg.SubRPCServers.RouterRPC.RoutingConfig.ProbabilityEstimatorType =
```

Comment:
This line is very long and the line break after the equal sign looks very weird.
How about:

```
	routerCfg := s.cfg.SubRPCServers.RouterRPC

	switch c := estimator.Config().(type) {
	case routing.AprioriConfig:
		routerCfg.ProbabilityEstimatorType =
			routing.AprioriEstimatorName

		targetCfg := routerCfg.AprioriConfig
		targetCfg.PenaltyHalfLife = c.PenaltyHalfLife
		targetCfg.Weight = c.AprioriWeight
		targetCfg.CapacityFraction = c.CapacityFraction
		targetCfg.HopProbability = c.AprioriHopProbability

	case routing.BimodalConfig:
		routerCfg.ProbabilityEstimatorType =
			routing.BimodalEstimatorName
		targetCfg := routerCfg.BimodalConfig
		targetCfg.Scale = int64(c.BimodalScaleMsat)
		targetCfg.NodeWeight = c.BimodalNodeWeight
		targetCfg.DecayTime = c.BimodalDecayTime
	}
	
```

Also, you seem to be mixing spaces and tabs. Make sure you run `make fmt` on each commit.

Reply:
I did the changes ... but a little mess on commits are back ... tomorrow morning I will fix it.

Reply:
I did just run the `make fmt` ... and fixed the commits ... Now I have done everything you pointed out. Thanks!